src/site/stages/build/process-cms-exports/transformers/node-event_listing.js

```diff
@@ -0,0 +1,56 @@
+const { getDrupalValue, utcToEpochTime } = require('./helpers');
+
+// Same as the createMetaTag in the transformer helpers, but uses
+// __typename instead of type. Because consistency.
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
+    key,
+    value,
+  };
+}
+
+const transform = entity => {
+  const metaTags = entity.metatag.value;
+
+  return {
+    entityType: 'node',
+    entityBundle: 'event_listing',
+    entityUrl: {
+      // TODO: Get the breadcrumb from the CMS export when it's available
+      breadcrumb: [],
+      path: entity.path[0].alias,
+    },
+    entityMetatags: [
+      createMetaTag('MetaValue', 'title', metaTags.title),
+      createMetaTag('MetaValue', 'twitter:card', metaTags.twitter_cards_type),
+      createMetaTag('MetaProperty', 'og:site_name', metaTags.og_site_name),
+      createMetaTag(
+        'MetaValue',
+        'twitter:description',
+        metaTags.twitter_cards_description,
+      ),
+      createMetaTag('MetaValue', 'description', metaTags.description),
+      createMetaTag('MetaValue', 'twitter:title', metaTags.twitter_cards_title),
+      createMetaTag('MetaValue', 'twitter:site', metaTags.twitter_cards_site),
+      createMetaTag('MetaProperty', 'og:title', metaTags.og_title),
+      createMetaTag('MetaProperty', 'og:description', metaTags.og_description),
+    ],
+    title: getDrupalValue(entity.title),
+    changed: utcToEpochTime(getDrupalValue(entity.changed)),
+    metatag: getDrupalValue(entity.metatag),
```

Comment:
This seems odd.  Do we actually need this?

Reply:
not sure where this is coming from. I don't see `metatag` in the transformed schema or the [test data](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/tests/entities/node.30295cab-89be-4173-a33c-8c0ca0a85d07.json). I get this error when I remove it. 

```
 1) CMS export node-event_listing should transform to match the test output:

      AssertionError: expected { Object (contentModelType, entityType, ...) } to deeply equal { Object (contentModelType, entityType, ...) }
      + expected - actual

             "entityLabel": "Outreach and events"
             "entityType": "node"
           }
         }
      +  "metatag": [null]
         "title": "Outreach events"
       }

      at Context.equal (src/site/stages/build/process-cms-exports/tests/assemble-entity-tree.unit.spec.js:77:39)
```